src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -617,6 +617,9 @@
     <None Include="UI\GuidedTour\dynamo_guides.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+	  <None Include="UI\GuidedTour\DynamoOnboardingGuide_HouseCreationDS.dyn">
+		  <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+	  </None>
```

Comment:
Despite the alignment issue here. @mjkkirschner Are you fine with this? I think this is hard copy way instead of embedded resources way you suggested?

Reply:
Hi, do we have a decision yet? 
For this, we will need to implement a loading function for the DYN file as it will be an embedded resource.

Reply:
hi @filipeotero We have never embedded a dyn in our resouces list before, what would be the impact on that in your opinion?

Reply:
@QilongTang I found in the framework of the open file that there is a `OpenFromJson()` function that facilitates the implementation here, having a low impact. I will commit the code soon. Thanks.